src/containers/legend/landcover-legend/styles.module.scss

```diff
@@ -0,0 +1,43 @@
+@import 'styles/settings';
+@import 'styles/typography-extends';
+
+.legendContainer {
+  bottom: 16px;
+  display: flex;
+  flex-direction: column;
+  height: 64px;
+  position: absolute;
+  left: $sidebar-width;
+  margin-left: 60px;
+  width: 640px;
```

Comment:
We have problems on smaller screens (in this case 1024px) Could you take a look? Maybe wrap to a second line